Point_set_processing_3/include/CGAL/structure_point_set.h

```diff
@@ -0,0 +1,1483 @@
+          }
+      }
+  }
+
+  void compute_corners (double radius)
+  {
+    if (m_edges.size () < 3)
+      return;
+
+    std::vector<std::vector<std::size_t> > plane_edge_adj (m_planes.size());
+    for (std::size_t i = 0; i < m_edges.size (); ++ i)
+      if (m_edges[i].active)
+        {
+          plane_edge_adj[m_edges[i].planes[0]].push_back (i);
+          plane_edge_adj[m_edges[i].planes[1]].push_back (i);
+        }
+
+    std::vector<std::set<std::size_t> > edge_adj (m_edges.size ());
+
+    for (std::size_t i = 0; i < plane_edge_adj.size (); ++ i)
+      {
+        if (plane_edge_adj[i].size () < 2)
+          continue;
+          
+        for (std::size_t j = 0; j < plane_edge_adj[i].size ()- 1; ++ j)
+          for (std::size_t k = j + 1; k < plane_edge_adj[i].size (); ++ k)
+            {
+              edge_adj[plane_edge_adj[i][j]].insert (plane_edge_adj[i][k]);
+              edge_adj[plane_edge_adj[i][k]].insert (plane_edge_adj[i][j]);
+            }
+      }
+
+    for (std::size_t i = 0; i < edge_adj.size (); ++ i)
+      {
+        if (edge_adj[i].size () < 2)
+          continue;
+
+        std::set<std::size_t>::iterator end = edge_adj[i].end();
+        end --;
+          
+        for (std::set<std::size_t>::iterator jit = edge_adj[i].begin ();
+             jit != end; ++ jit)
+          {
+            std::size_t j = *jit;
+            if (j < i)
+              continue;
+
+            std::set<std::size_t>::iterator begin = jit;
+            begin ++;
+            for (std::set<std::size_t>::iterator kit = begin;
+                 kit != edge_adj[i].end (); ++ kit)
+              {
+                std::size_t k = *kit;
+                if (k < j)
+                  continue;
+                  
+                std::set<std::size_t> planes;
+                planes.insert (m_edges[i].planes[0]);
+                planes.insert (m_edges[i].planes[1]);
+                planes.insert (m_edges[j].planes[0]);
+                planes.insert (m_edges[j].planes[1]);
+                planes.insert (m_edges[k].planes[0]);
+                planes.insert (m_edges[k].planes[1]);
+
+                if (planes.size () == 3) // Triple found
+                  {
+                    std::vector<std::size_t> vecplanes (planes.begin (), planes.end ());
+                    m_corners.push_back (Corner (vecplanes[0], vecplanes[1], vecplanes[2],
+                                                 i, j, k));
+                  }
+              }
+          }
+      }
+
+
+    for (std::size_t i = 0; i < m_corners.size (); ++ i)
+      {
+        //calcul pt d'intersection des 3 plans
+        Plane plane1 = static_cast<Plane> (*(m_planes[m_corners[i].planes[0]]));
+        Plane plane2 = static_cast<Plane> (*(m_planes[m_corners[i].planes[1]]));
+        Plane plane3 = static_cast<Plane> (*(m_planes[m_corners[i].planes[2]]));
+        Line line;
+
+        CGAL::Object ob_temp = CGAL::intersection(plane1, plane2);
```

Comment:
btw,  you should be aware that most of the time the kernel used is inexact. You should check whether this is an issue or not.